Directory.Build.targets

```diff
@@ -19,6 +19,10 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetPlatformIdentifier)' != 'ios' And '$(TargetPlatformIdentifier)' != 'maccatalyst'">
+    <!-- Exclude our platform-specific code: -->
+    <Compile Remove="**\*.Cocoa.cs" />
+    <Compile Remove="**\Cocoa\**\*.cs" />
+    <!-- Maui device test projects have these too: -->
     <Compile Remove="**\*.iOS.cs" />
     <Compile Remove="**\iOS\**\*.cs" />
```

Comment:
Not sure I get those. Do you mean the Maui tests?

Reply:
Yes